docs/tutorial/upgrading-kubernetes.md

```diff
@@ -2,16 +2,19 @@
 
 Upgrading kubernetes is very easy with kops, as long as you are using a compatible version of kops.
 The kops `1.8.x` series (for example) supports the kubernetes 1.6, 1.7 and 1.8 series,
-as per the kubernetes deprecation policy.  Older versions of kubernetes will likely still work, but these
-are on a best-effort basis and will have little if any testing.  kops `1.8` will not support the kubernetes
-`1.9` series, and for full support of kubernetes `1.9` it is best to wait for the kops `1.9` series release.
+as per the kubernetes deprecation policy.
+
+Older versions of kubernetes will likely still work, but these
+are on a best-effort basis and will have little if any testing.  kops `1.9` will not support the kubernetes
+`1.10` series, and for full support of kubernetes `1.10` it is best to wait for the kops `1.10` series release. See the [compatibility matrix](../../README.md#compatibility-matrix).
+
 We aim to release the next major version of kops within a few weeks of the equivalent major release of kubernetes,
-so kops `1.9.0` will be released within a few weeks of kubernetes `1.9.0`.  We try to ensure that a 1.9 pre-release
+so kops `1.10.0` will be released within a few weeks of kubernetes `1.10.0`.  We try to ensure that a 1.10 pre-release
 (alpha or beta) is available at the kubernetes release, for early adopters.
 
 Upgrading kubernetes is similar to changing the image on an InstanceGroup, except that the kubernetes version is
 controlled at the cluster level.  So instead of `kops edit ig <name>`, we `kops edit cluster`, and change the
-`kubernetesVersion` field.  `kops edit cluster` will open your editor with the cluster, similar to:
+`kubernetesVersion` field.  `kops edit cluster` will open your editor with the cluster spec loaded, similar to this with a kubernetesVersion of 1.9.0:
```

Comment:
s/will open your editor with the cluster spec loaded, similar to this with a kubernetesVersion of 1.9.0/opens your current cluster spec in an editor. This is an example of the field to edit: